crates/cairo-profiler/src/cli/mod.rs

```diff
@@ -0,0 +1,25 @@
+use crate::cli::build_profile::BuildProfile;
+use crate::cli::view::ViewProfile;
+use clap::{Parser, Subcommand};
+
+pub(crate) mod build_profile;
+pub(crate) mod view;
+
+#[derive(Parser)]
+#[command(version, args_conflicts_with_subcommands = true)]
+#[clap(name = "cairo-profiler")]
+pub struct Cli {
+    #[clap(flatten)]
+    pub build_profile_args: Option<BuildProfile>,
```

Comment:
```suggestion
    pub build_profile_args: BuildProfile,
```

Reply:
What are the upsides of this potentially? I kinda like it being explicitly stated that this is Optional, but am ready to change my mind :D

Reply:
After some research i come to conclusion that marking this with option is better solution. I even made PR to fix this in coverage:

https://github.com/software-mansion/cairo-coverage/pull/124